### ISSUE TITLE
Pin `customer_testing` for release branches

### ISSUE THREAD

**Issue by christopherfujino** [flutter/flutter#57062]
`customer_testing` should be pinned to the version of the repo at the time a release branched from master.

**Comment by christopherfujino** [comment]
I discussed this in the Infra meeting, and we agreed that infra scripts written in Dart should use a stable release, packaged with CIPD.

**Comment by christopherfujino** [comment]
@nturgut was adding a pinned version of the framework to the engine DEPS file considered? Is that out of the question?

**Comment by nturgut** [comment]
No, we haven't considered that approach for flutter -> engine dependency.

We are using a pinned version of flutter/web_installers inside flutter/engine though

**Comment by christopherfujino** [comment]
One possible world would be both the Framework and the Engine use gclient to fetch all CI-time dependencies, with pinned hashes. Then we don't need to write any scripts to clone/checkout particular hashes, and all the pinned hashes are in a known place.

**Comment by itsjustkevin** [comment]
@christopherfujino this issue is rather old.  Is it still relevant today?

**Comment by christopherfujino** [comment]
> @christopherfujino this issue is rather old. Is it still relevant today?

I'll close this, as I'm not gonna work on it.
